server/middleware/github-griller/index.js

```diff
@@ -0,0 +1,43 @@
+let repos = require('./github.repos');
```

Comment:
@ssarda77 These different parts of the module are all imported here.  They then can be passed in a dependencies objection, 'dep', into whichever functions need them.  eg, getAllRepos needs to use utils, so it's passed in an object.  getOneRepo needs a few more, and so they're all passed in.